### PR TITLE
Differentiate Between Communication Towers and Masts

### PR DESCRIPTION
**Pull request by Nakaner**

### CODE REVIEW

File: josm-presets/de.xml [hunk 1]
```diff
@@ -1707,13 +1707,29 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
...
 				value="phone" />
 		</item>
-		<item name="Zugfunk" type="node">
-			<label text="Zugfunk" />
+		<item name="Radio tower/mast" de.name="Zugfunkmast" type="node">
+			<label text="Radio tower/mast" de.text="Zugfunkmast" />
 			<space />
 			<key key="railway"
 				value="radio" />
 			<key key="man_made"
```
Comment (rurseekatze) [inline josm-presets/de.xml]: You should remove the fixed man_made=mast since you added a combo box.

Comment (Nakaner) [inline josm-presets/de.xml]: I have fixed this and rebased onto master.

**Comment by rurseekatze** [comment]
This differentiation has to be documented in the wiki pages too.

**Comment by Nakaner** [comment]
rurseekatze wrote:
> This differentiation has to be documented in the wiki pages too.

Done.

**Pull request merged**